template.js

```diff
@@ -103,37 +92,77 @@ if (fbc) {
 if (fbp) {
   setCookie('_fbp', fbp, cookieOptions);
 }
-sendHttpRequest(
-  postUrl,
-  (statusCode, headers, body) => {
+
+const apiVersion = '18.0';
+const postUrl = 'https://graph.facebook.com/v' + apiVersion + '/' + enc(data.pixelId) + '/events?access_token=' + enc(data.accessToken);
+
+const pixelIdsAndAccessTokens = [{ pixelId: data.pixelId, accessToken: data.accessToken }];
+if (data.enableMultipixelSetup) {
+  pixelIdsAndAccessTokens.concat(data.pixelIdAndAccessTokenTable);
```

Comment:
@giovaniortolani now tag sends only one request even if  ```enableMultipixelSetup``` enabled, because ```pixelIdsAndAccessTokens``` is always an array of 1 object.
```pixelIdsAndAccessTokens``` should be re-assigned because concat returns a new array.

Reply:
Sorry, @kHorozhanov. I ended up pasting the wrong version.
Just sent the correct version now.